### DISCUSSION TITLE
Change default list call in grid

### DISCUSSION THREAD

**Discussion by EdwinThomas11** [serenity-is/Serenity#6994] [Help] [answered]
Hello, 

I would like to change the default list call which is triggered to populate the records in the grid. Please help.

Thanks

**Comment by EdwinThomas11** [comment]
@volkanceylan  @dfaruque @VictorTomaili  appreciate your inputs here

**Comment by EdwinThomas11** [comment]
I have two screen which uses the same row file but different form file and column file. For first screen I would like to use a list call and for the second screen i would like to use another api (like customlist). I am using two different grid and dialog ts files for both the screens. Is i possible to override the default list call from the grid?

**Comment by EdwinThomas11** [comment]
I attempted to establish separate endpoints for each screen, and this approach proved successful. However, I'm interested in learning whether it's feasible to override the default list call within the dialog file.

**Comment by luongnv6** [comment, marked as answer]
try
```ts
        protected getViewOptions(): Slick.RemoteViewOptions {
            var opt = super.getViewOptions();
            opt.url = Q.resolveUrl('~/Services/' + yourEndPoint); //default '~/Services/List'
            return opt;
        }
```

**Reply by EdwinThomas11** [reply]
Thanks @luongnv6. This works
